nnstreamer.pc.in

```diff
@@ -9,5 +9,5 @@ Name: nnstreamer
 Description: Custom Plugin Dev Kit of Neural Network Suite for GStreamer
 Version: @VERSION@
 Requires:
-Libs:
+Libs: L${libdir} -lnnstreamer
```

Comment:
In addition to libdir, we may need to add gstdir as well.?

Reply:
@myungjoo  hmm.... IMO, app developers would be OK with libdir and plugin developers would add build dependencies on Gstreamer-devel so that they also do not need gstdir here. I am not sure.

Reply:
You specified ```-lnnstreamaer```, which is installed in gstdir, not libdir.
In libdir, we have ```*.a``` files and subplugins.

Reply:
@myungjoo You are right. It would be modified.